WcaOnRails/config/locales/es.yml

```
@@ -1491,6 +1491,7 @@ es:
       manage_app: Controlar tus aplicaciones
       #original_hash: 0e8b212
       manage_auth_app: Controlar aplicaciones autorizadas
+      media_submission: "Entrega de medios"
```

@Ivan-Brigidano could we please get a quick review on that translation?

---

IMO this sounds way better:
```suggestion
      media_submission: "Enviar material audiovisual"
```